lib/spl-build.js

```diff
@@ -423,33 +441,50 @@ export default {
   },
 
   shouldShowMenuSpl(event) {
+    if (atom.workspace.getActivePane().activeItem.constructor.name === 'TextEditor') {
```

Comment:
```suggestion
    if (atom.workspace.getActiveTextEditor()) {
```

Per the atom api docs, getActiveTextEditor() will return undefined if the center panel is not an editor. So we can use that as our test here, safer way to do it. Please update the other occurrences with this change.

https://atom.io/docs/api/v1.38.2/Workspace#instance-getActiveTextEditor